testsuite/features/support/env.rb

```diff
@@ -14,12 +14,20 @@
 require 'selenium-webdriver'
 require 'multi_test'
 require 'set'
+require_relative 'code_coverage'
 
 ## code coverage analysis
 # SimpleCov.start
```

Comment:
I know this is not covered by your PR, but do we still need this? Same above in line 11.

Reply:
That's different, this is code coverage for the test code itself.
For now, let's keep it there to don't mix concepts.